nodes/viz/texture_viewer.py

```diff
@@ -284,10 +286,16 @@ def save_bitmap(self, operator):
         print(img_format)
         if img_format == 'TARGA':
             extension = '.' + img_format.lower().replace('targa', 'tga')
+        elif img_format == 'TARGA_RAW':
+            extension = '.' + img_format.lower().replace('targa_raw', 'tga')
         elif img_format == 'JPEG2000':
             extension = '.' + img_format.lower().replace('jpeg2000', 'jp2')
         elif img_format == 'OPEN_EXR':
             extension = '.' + img_format.lower().replace('open_exr', 'exr')
+        elif img_format == 'OPEN_EXR_MULTILAYER':
+            extension = '.' + img_format.lower().replace('open_exr_multilayer', 'exr')
+        elif img_format == 'IRIS':
+            extension = '.' + img_format.lower().replace('iris', 'rgb')
```

Comment:
you are right, but i started with one more format added and ended with a lot of other...! :smile: